checkmk_extensions/agent/scheduler_runner.ps1

```diff
@@ -56,3 +56,87 @@ class Command {
         return [Command]::new($_executable, $_argumentsList)
     }
 }
+
+function StartSchedulerRunner {
+    [CmdletBinding()]
+    param (
+        [Parameter(Mandatory=$true)]
+        [string]$configFilePath
+    )
+
+    $configFileContent = [Config]::ParseConfigFile($configFilePath)
+
+    $command = [Command]::CreateCommand($configFileContent)
+    $parentDir = Split-Path $PSScriptRoot -Parent
+    $logPath = Join-Path $parentDir "log\robotmk\scheduler_runner.log"
+
+    # Create processInfo
+    $processInfo = New-Object System.Diagnostics.ProcessStartInfo
+    $processInfo.FileName = $command.Executable
+    $processInfo.Arguments = $command.ArgumentsList
+    $processInfo.RedirectStandardOutput = $true
+    $processInfo.RedirectStandardError = $true
+    $processInfo.UseShellExecute = $false
+    $processInfo.CreateNoWindow = $true
+    # $processInfo.WorkingDirectory -> Using this we can configure from where the Process will be started
+    $processInfo.WorkingDirectory = $PSScriptRoot
+
+    # Create process and assign processInfo to it
+    $process = New-Object System.Diagnostics.Process
+    $process.StartInfo = $processInfo
+
+
+    try {
+        $process.Start() | Out-Null
```

Comment:
This should happen inside the loop. If the process crashes, we want to re-start it (should never happen, of course).